layer/mounted_layer.go

```diff
@@ -50,11 +53,68 @@ func (ml *mountedLayer) Parent() Layer {
 	return nil
 }
 
+func (ml *mountedLayer) incActiveMount() {
```

Comment:
We now have `incActiveMount` and `incActivityCount` methods. Would it be simpler in `ReinitRWLayer` to cast to `referenceLayer` and call `acquire()` directly, skipping the extra method. It's should also be safer than string based lookup from map.